### PR TITLE
Add Syntax Highlighting for Snippets

### PR DESCRIPTION
**Pull request by msaglietto**
Use Code Mirror lib on code editor component that will display docker file and shell test code

This connects to #20

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -5,12 +5,13 @@
 
                  [javax.servlet/servlet-api "2.5"] [ring "1.3.2"] [compojure "1.5.0"] ;;to be able to run compojure as a standalone jar
 
-                 [reagent "0.5.1"]                          ; React rendering wrapper
+                 [reagent "0.6.0-alpha2"]                   ; React rendering wrapper
```
Comment (Volland) [inline project.clj]: what the reason to use alpha ?

File: resources/public/index.html [hunk 2]
```diff
@@ -19,6 +21,10 @@
       <!--<div class="signal"/>-->
     </div>
     <script src="js/compiled/app.js"></script>
+    <!-- Needed by code-editor component TODO: Make load on demand by the component -->
```
Comment (Volland) [inline resources/public/index.html]: Could we use pre build  minimised version as a single js file  ?

File: src/cljs/pazuzu_ui/registry/view.cljs [hunk 3]
```diff
@@ -48,16 +50,10 @@
               [:i.add.icon] "Add"]]]]
           [:div.field.code
            [:label "Docker file Snippet"]
-           [:textarea {:field     :textarea
-                       :rows      5
```
Comment (Volland) [inline src/cljs/pazuzu_ui/registry/view.cljs]: Could we extract magic numbers to config or constants ?

File: project.clj (hunk 1 above)
Comment (msaglietto) [inline project.clj]: Because 0.6 use React 15 that bring nice improvements

File: resources/public/index.html (hunk 2 above)
Comment (msaglietto) [inline resources/public/index.html]: I can see how the build process works and create it .. but if you know of it go ahead and build the bundle

File: src/cljs/pazuzu_ui/registry/view.cljs (hunk 3 above)
Comment (msaglietto) [inline src/cljs/pazuzu_ui/registry/view.cljs]: Is not a magic number is a html attribute and was removed =P

**Comment by Volland** [comment]
👍

**Pull request merged**